docs/dump_systemdump.md

```diff
@@ -0,0 +1,41 @@
+<!--
+* Copyright (c) 2017, 2018 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# System dump
+
+System dumps, often known as *core dumps*, are platform-specific and contain a raw binary dump of the process memory. This type of dump has a complete copy of the Java heap, including the contents of all Java objects in the application.
+
+To examine a system dump you can use the OpenJ9 dump viewer (`jdmpview`) or a platform-specific debugging tool.
+
+<i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Note:** On Linux&trade; and AIX&reg; systems you must use the OpenJ9 dump extractor (`jextract`) to collect additional files for analysis before you can view the dump.
```

Comment:
This is no longer accurate. There is no requirement to use jextract before using jdmpview (note in the IBM doc the statement about using jextract is qualified with "if the system dump was generated by a version of the J9 virtual machine before V2.6").

The only purpose for jextract for OpenJ9 is to collect system libraries on LKinux or AIX when engaging the IBM service organization. https://www.ibm.com/support/knowledgecenter/SSYKE2_8.0.0/com.ibm.java.vm.80.doc/docs/systemdump_viewer_jextract.html

Reply:
I've removed the note about jextract, and the topic in the Tools section. Now I'm looking at the Dump viewer topic and seeing the need for some changes there too (in the IBM doc aswell). I can remove references here to jextract, but should I also remove all the references to passing in "zip" on the command line when running jdmpview. There a quite a few of these, so checking first.

Reply:
Done. I also removed -notemp as that also seemed redundant judging by the description.